### PR TITLE
Two similar types for e-mail address

### PR DESCRIPTION
**Pull request by trurlurl**
Removed `EmailAddressType` in OJP_FareSupport.xsd - there is already `EMailAddressType`in OJP_Utility.xsd.

### CODE REVIEW

File: OJP/OJP_FareSupport.xsd [hunk 1]
```diff
@@ -176,7 +170,7 @@
 					<xs:documentation>Name of contact person.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="Email" type="EmailAddressType" minOccurs="0">
+			<xs:element name="EMail" type="EMailAddressType" minOccurs="0">
```
Comment (skinkie) [inline OJP/OJP_FareSupport.xsd:173]: Any reason why you prefer EMail with capital M?

Comment (trurlurl) [reply OJP/OJP_FareSupport.xsd:173]: @skinkie Yes, because for me e-mal is composed of two words (e/electronic and mail).

Comment (skinkie) [reply OJP/OJP_FareSupport.xsd:173]: In a serious language yes, but not in English. https://www.merriam-webster.com/dictionary/email

Comment (trurlurl) [reply OJP/OJP_FareSupport.xsd:173]: Haha, well, but it also seems to be a bit a matter of taste. If more people prefer Email to EMail, I wouldn't oppose changing it.

Comment (ue71603) [reply OJP/OJP_FareSupport.xsd:173]: I suggest we go with the webster definition.

**Review by sgrossberndt — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**